File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/convertMyOptionsToString.cpp
#include "convertMyOptionsToString.h"
#include <string>
#include "SelectEntryCommand_ViewModelMyOptionsOption.h"

namespace commands
{
  std::string convertMyOptionsToString(SelectEntryCommand_ViewModelMyOptionsOption symbol)
  {
    if (symbol == commands::SelectEntryCommand_ViewModelMyOptionsOption::A)
    {
      return std::string("A");
    }

    if (symbol == commands::SelectEntryCommand_ViewModelMyOptionsOption::B)
    {
      return std::string("B");
    }

    return std::string("?");
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/ConvertMyOptionsToString.cs
/// <filename>
///     ConvertMyOptionsToString.cs
/// </filename>
using commands;

namespace commands
{
    public class ConvertMyOptionsToString
    {
        public static string Execute(commands.SelectEntryCommand_ViewModelMyOptionsOption symbol)
        {
            if (symbol == commands.SelectEntryCommand_ViewModelMyOptionsOption.A)
            {
                return "A";
            }

            if (symbol == commands.SelectEntryCommand_ViewModelMyOptionsOption.B)
            {
                return "B";
            }

            return "?";
        }
    }
}
